### PR TITLE
Created fast_corner.py

### PR DESCRIPTION
**Pull request by adeel-20**
### Describe your change:

* [x] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### Checklist:
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms include at least one URL that points to Wikipedia or another similar explanation.
* [x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: computer_vision/fast_corner.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+Features from accelerated segment test (FAST) Corner Detector
+https://en.wikipedia.org/wiki/Features_from_accelerated_segment_test
+"""
+
+
+def fast_detector(img):
+    fast = cv.FastFeatureDetector_create()
+    """
+    Drawing keypoints
+    """
```
Comment (CaedenPH) [inline computer_vision/fast_corner.py:13]: These are comments, and should be indicated with the python comment token `#`
```suggestion
    # Drawing keypoints
```

File: computer_vision/fast_corner.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    # Print all default params
+    print(f"Threshold: {fast.getThreshold()}")
+    print(f"nonmaxSuppression:{fast.getNonmaxSuppression()}")
+    print(f"neighborhood: {fast.getType()}")
+    print(f"Total Keypoints with nonmaxSuppression: {len(kp)}")
+    cv.imwrite("fast_true.png", img2)
+    """
+    NON_MAX(NMS) Supression Disabled
+    """
+    fast.setNonmaxSuppression(0)
```
Comment (CaedenPH) [inline computer_vision/fast_corner.py:25]:
```suggestion
    fast.setNonmaxSuppression(0)  # NON_MAX(NMS) Supression Disabled
```
Similiar to above - 
Is this comment even necessary though?

File: computer_vision/fast_corner.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
+import cv2 as cv
+
+"""
+Features from accelerated segment test (FAST) Corner Detector
+https://en.wikipedia.org/wiki/Features_from_accelerated_segment_test
+"""
```
Comment (CaedenPH) [inline computer_vision/fast_corner.py:6]:
```suggestion
"""
Features from accelerated segment test (FAST) Corner Detector
https://en.wikipedia.org/wiki/Features_from_accelerated_segment_test
"""

import cv2 as cv
```

File: computer_vision/fast_corner.py [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+    """
+    Drawing keypoints
+    """
+    kp = fast.detect(img, None)
+    img2 = cv.drawKeypoints(img, kp, None, color=(255, 0, 0))
+    # Print all default params
+    print(f"Threshold: {fast.getThreshold()}")
+    print(f"nonmaxSuppression:{fast.getNonmaxSuppression()}")
+    print(f"neighborhood: {fast.getType()}")
+    print(f"Total Keypoints with nonmaxSuppression: {len(kp)}")
```
Comment (CaedenPH) [inline computer_vision/fast_corner.py:20]: In my opinion , it is preferable for the algorithms to not print any output and instead just return data

File: computer_vision/fast_corner.py (hunk 1 above)
Comment (adeel-20) [reply computer_vision/fast_corner.py:13]: Noted, will update :)

File: computer_vision/fast_corner.py (hunk 4 above)
Comment (adeel-20) [reply computer_vision/fast_corner.py:20]: Noted, will remove them.

File: computer_vision/fast_corner.py (hunk 2 above)
Comment (adeel-20) [reply computer_vision/fast_corner.py:25]: I guess not, will update. Thanks for the corrections.

**Pull request closed without merging**